.github/actions/make_init/action.yml

```diff
@@ -15,8 +15,12 @@ runs:
       uses: actions/setup-node@v4
       with:
         node-version-file: ".nvmrc"
-        cache: "yarn"
```

Comment:
Assuming approval works, I will re-enable this cache and clear the caches for actions so that this can replace it and merge. It will be a temporary blocker for people, but one that is easy to fix.